knack/commands.py

```diff
@@ -99,6 +99,7 @@ def _resolve_default_value_from_config_file(self, arg, overrides):
             logger.info("Configured default '%s' for arg %s", config_value, arg.name)
             overrides.settings['default'] = DefaultStr(config_value)
             overrides.settings['required'] = False
+            overrides.settings['default_from'] = 'Configure'
```

Comment:
What other else sources could be ?

Reply:
For local context, it's `Local Context`. If no value for this property, it will be treated as `Default`.

Reply:
What other values could be?

Reply:
Currently, there are only 3 sources: Default, Configure, Local Context.